app/services/rating_calculator.rb

```diff
@@ -0,0 +1,13 @@
+class RatingCalculator
```

Comment:
please change to a module with calculate method that returns the value
Class here can stay in memory and hold reviews for no reason